static/src/js/utils/parseRdfDatatoInitialData.js

```diff
@@ -0,0 +1,127 @@
+const parseRdfDataToInitialData = (rdfData) => {
```

Comment:
Hoan just included imported fast-xml-parser (which we use in kms), that might have made the processing easier.   Than you are just dealing with json.